spec/pronto/formatter/checkstyle_formatter_spec.rb

```diff
@@ -6,20 +6,34 @@ module Formatter
       describe '#format' do
         subject { formatter.format(messages, nil, nil) }
         let(:line) { double(new_lineno: 1, commit_sha: '123') }
-        let(:error) { Message.new('path/to', line, :error, 'Line Error') }
-        let(:warning) { Message.new('path/to', line, :warning, 'Line Warning') }
+        let(:error) do
+          Message.new(path:  'path/to',
+                      line:  line,
+                      level: :error,
+                      msg:   'Line Error')
```

Comment:
Please use this style of alignment (consistent in project, easier to change, does not require editor plugins :smile:):

```ruby
Message.new(path: 'path/to',
            line: line,
            level: :error,
            msg: 'Line Error')
```